src/pages/class/[cid]/question/[qid]/solve.tsx

```diff
@@ -121,56 +122,81 @@ export default function Page() {
   )
 
   return (
-    <Sheet gap={0}>
-      <Label color="grey200" marginBottom={8}>
-        Contributors
-      </Label>
-      <Contributors>
-        {contributors?.userData?.map(
-          (contributor, index: number) =>
-            contributor.img && <ProfileImg src={contributor.img} key={index} title={contributor.name}></ProfileImg>
-        )}
-      </Contributors>
-      <Divider marginVertical={20} />
-      <Label color="grey200" marginBottom={16}>
-        Q. {qinfo?.stem_text}
-      </Label>
-      <div>
-        {optionSet?.map((e, i) => {
-          return (
-            <OptionButton
-              onClick={() => {
-                setSelected(i)
-              }}
-              state={0 <= selected}
-              selected={selected === i}
-              key={i}
-              isAnswer={showAnswer && answer === i}
-              marginBottom={8}
-            >
-              {e.option_text}
-            </OptionButton>
-          )
-        })}
-      </div>
-      <BtnDisplay>
-        <FillButton
-          onClick={() => {
-            checkAnswer()
-            setShowAnswer(true)
-          }}
-          disabled={selected == -1}
-        >
-          Submit
-        </FillButton>
-        <StrokeButton onClick={shuffleOptions}>Shuffle Options</StrokeButton>
-        <StrokeButton onClick={toggleModal}>Report Error</StrokeButton>
-        <InputDialog modalState={isOpenModal} submit={reportSubmit} toggleModal={toggleModal} />
-      </BtnDisplay>
-    </Sheet>
+    <Wrapper>
```

Comment:
you can do:
```suggestion
    <>
        <Sheet gap={0} maginBottom={10}>
        ...
        {showAnswer && (
        ...
        )}
    </>
```

Reply:
let's reduce the number of new custom components if possible

Reply:
> let's reduce the number of new custom components if possible

Sure thing, updated here https://github.com/kixlab/kuiz/pull/80/commits/b5eb2d0abe4985088e06226446443f0dfe64d446 :)